nengo_viz/static/viz.js

```diff
@@ -4,10 +4,21 @@ var VIZ = {};
 VIZ.max_zindex = 0;
 
 /**
+ * Helper function to clip a number, keeping it between two values.
+ */
+VIZ.clip = function(x, low, high) {
+    if (x < low)
```

Comment:
Should always include the `{ }` even if they're not necessary; it's too easy to add another line to this and think that it's in the body of the if when it's actually not.